src/udisksdaemonutil.c

```diff
@@ -904,48 +958,18 @@ udisks_daemon_util_get_caller_uid_sync (UDisksDaemon            *daemon,
                                         GError                 **error)
 {
   gboolean ret;
-  const gchar *caller;
-  GVariant *value;
-  GError *local_error;
   uid_t uid;
 
   /* TODO: cache this on @daemon */
 
   ret = FALSE;
 
-  caller = g_dbus_method_invocation_get_sender (invocation);
-
-  local_error = NULL;
-  value = g_dbus_connection_call_sync (g_dbus_method_invocation_get_connection (invocation),
-                                       "org.freedesktop.DBus",  /* bus name */
-                                       "/org/freedesktop/DBus", /* object path */
-                                       "org.freedesktop.DBus",  /* interface */
-                                       "GetConnectionUnixUser", /* method */
-                                       g_variant_new ("(s)", caller),
-                                       G_VARIANT_TYPE ("(u)"),
-                                       G_DBUS_CALL_FLAGS_NONE,
-                                       -1, /* timeout_msec */
-                                       cancellable,
-                                       &local_error);
-  if (value == NULL)
+  if (!dbus_freedesktop_guint32_get(invocation, cancellable, "GetConnectionUnixUser",
+                   &uid, error))
```

Comment:
Please add a space after the function name and indent the follow-up line under the first parameter.